tool/speed.cc

```diff
@@ -113,6 +120,19 @@ struct TimeResults {
     }
   }
 
+  void PrintWithPrimes(const std::string &description,
```

Comment:
Would it be interesting to print out some of the other statistics like  min/mediian/max/std deviation?